README.md

```diff
@@ -124,3 +133,105 @@ Path parameters
 |:----------|:-----|:---------|
 | ECNumber | String | The EC number you want ligands associated with |
+
+
+#### Docking
+
+###### To perform docking, start by calling the ```POST /dock/dockligand``` endpoint.
+###### Once you receive a 200 status and a job ID, continue to query the ```GET /dock/dockligand``` endpoint
+###### Once you receive a 200 status, active site data, binding affinities and job ID, call ```GET /dock/retrievefile```
+###### Continue to call ```GET /dock/retrievefile``` until a 200 status is returned.
+###### If you would like to receive the completed docked file again, you may call the ```GET /dock/retrievefile``` endpoint
+###### The location of the AutoDock Vina and OpenBabel deployments may be configured in the ```application.properties``` file
+
+#### Docking API
+
+<a name="dock-ligand"></a>
+##### POST /dock/dockligand
+###### Convert the supplied ligand and protein to pdbqt format using OpenBabel then start an AutoDock Vina job
+
+##### Form Parameters
+
+| Parameter | Type | Function |
+|:----------|:-----|:---------|
+| macromolecule | File | Protein file for docking |
+| ligand | File | Ligand file for docking |
+| ligandID | String | Optional ligand ID. If ligand file is not provided, ligand ID will be used to download ligand from the PDB |
+| macromoleculeID | String | Optional macromolecule ID. If macromolecule file is not provided, macromolecule ID will be used to download macromolecule from the PDB |
+| center_x | number | Center of docking area in the x axis |
+| center_y | number | Center of docking area in the y axis |
+| center_z | number | Center of docking area in the z axis |
+| size_x | number | Size of docking area in the x axis |
+| size_y | number | Size of docking area in the y axis |
+| size_z | number | Size of docking area in the z axis |
+
+##### Returns
+
+###### 200 OK - When job has been completed successfully
+
+| Parameter | Type | Function |
+|:----------|:-----|:---------|
+| jobId | String | ID of the AutoDock Vina job |
+| macromolecule | String | Name of macromolecule file |
+| ligand | String | Name of ligand file |
+
+<a name="get-docked-ligand"></a>
+##### GET /dock/dockligand
+###### Checks the status of an AutoDock Vina job. When complete, sends docked ligand file to OpenBabel and returns a list of binding affinities and active sites.
+###### For active site generation to function, ensure that the motif database has been generated.
+
+##### Path Parameters
+
+| Parameter | Type | Function |
+|:----------|:-----|:---------|
+| jobId | String | ID of the AutoDock Vina job |
+| pdbId | String | Id of protein for determination of active sites |
+
+##### Returns
+
+###### 200 OK - When job has completed successfully
+###### 203 - When job is still processing
+
+| Parameter | Type | Function |
+|:----------|:-----|:---------|
+| babelJobId | String | Job ID of OpenBabel file conversion |
+| dockingData | array | List of binding affinities and free energy values |
```

Comment:
something indicating how to interpret the docking data would be helpful - maybe just something indicating what value is in each array position

Reply:
I have added a guide for interpreting the docking data as well as a table for the active site data JSON objects.